spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/JedisConnectionConfiguration.java

```diff
@@ -95,6 +96,19 @@ private JedisClientConfigurationBuilder applyProperties(JedisClientConfiguration
 		return builder;
 	}
 
+	private void createPoolingBuilder(JedisClientConfigurationBuilder builder, Environment environment, Pool pool) {
+		final boolean poolEnabled = environment.getProperty("spring.redis.jedis.pool.enabled", Boolean.class, true);
+		if (poolEnabled) {
+			applyPooling(pool, builder);
+			return;
+		}
+		final boolean isSentinelConfig = (getSentinelConfig() != null);
+		// Jedis Sentinel cannot operate without a pool.
+		if (isSentinelConfig) {
+			throw new RedisClientPoolingException(ClientType.JEDIS);
```

Comment:
That looks like something new to me and unrelated to configure the pool automatically. I am happy to consider this case at a later stage but for now please revert.

Reply:
I got it. Regarding this, I will report it in another issue or pr.